runtime/service.go

```diff
@@ -814,7 +815,7 @@ func (s *service) Delete(requestCtx context.Context, req *taskAPI.DeleteRequest)
 // Exec an additional process inside the container
 func (s *service) Exec(requestCtx context.Context, req *taskAPI.ExecProcessRequest) (*ptypes.Empty, error) {
 	defer logPanicAndDie(log.G(requestCtx))
-	logger := s.logger.WithField("TaskID", req.ID).WithField("ExecID", req.ExecID)
+	logger := s.logger.WithField("id", req.ID).WithField("exec_id", req.ExecID)
```

Comment:
I'm good with this update in general as I agree the logs have grown inconsistent as the code evolved over time. But I would prefer that they consistently use `TaskID` and `ExecID` (or `task_id` and `exec_id`, don't have a preference on the casing style) as I think that's the most explicit names for the fields.

Reply:
Also, it may be worth just defining these field name strings as constants somewhere so their definition can just be centralized. I don't think that's absolutely necessary though, just something to consider.

Reply:
Thanks. Will use `task_id` instead of `id` then.

Regarding the constants, I would like to have helper methods. Maybe logForTask and logForExec?

Reply:
Yeah helper methods are a good idea too

Reply:
Actually we are sometimes takes a logger from the context passed-in (`log.G(context)`) and sometimes takes a logger from the service instance. I'm going to rename "id" to "task_id" in this pull request, and will add helper methods to create a logger consistently.